web/app_options.js

```diff
@@ -454,7 +459,6 @@ const defaultOptions = {
         : "../web/wasm/",
     kind: OptionKind.API,
   },
-
```

Comment:
This line-break should not be removed, since it helps readability by separating API and WORKER options.